swagger.yaml

```diff
@@ -252,9 +248,13 @@ definitions:
         enum:
```

Comment:
note:
added new states to swagger spec after the release branch was created - will need to merge back to develop.